frontend/components/product_markup.js

```diff
@@ -0,0 +1,46 @@
+/*
+<div key={ i } className='products-summary'>
+    <div className='products-summary__name'>
+    { datum.name }
+    </div>
+    <div className='products-summary__brand'>
+    <i className='products-summary__brand-by italic'> by </i> { datum.brand }
+    </div>
+    <div className='products-summary__data-input-status'>
+    <i className='italic'> Status </i> { datum.dataInputStatus }
+    </div>
+</div>
+*/
+
+import React, { PureComponent } from 'react';
+
+export default class ProductMarkup extends PureComponent {
+  productName = null;
+  productBrand = null;
+  productStatus = null;
+  state = {};
+
+  render() {
+    const {
+      productName,
```

Comment:
you should pass the entire product object as 1 prop so you can access all of it's fields without having to update the component every time you need a new field. This is a good rule of thumb to follow, when passing db models around, just pass the entire thing and call it 'modelName' + data, so maybe `productData={...}`